src/dynamic_sharing_service/policy/dynamic.policy.builder.js

```diff
@@ -30,8 +32,17 @@ module.exports = class DynamicPolicyBuilder {
     if (wantedAttribute.getDerivation()) {
       key = wantedAttribute.getDerivation();
     }
+    if (wantedAttribute.getConstraints()) {
+      const hash = crypto
+        .createHash('sha256')
+        .update(JSON.stringify(wantedAttribute.getConstraints()), 'utf8')
+        .digest('hex');
+
+      key = `${key}:${hash}`;
```

Comment:
Super pedantic, but is it worth keeping this the same as the other SDK's where I think we use `-` instead of `:`?

Reply:
Changed in d1ec770